### PR TITLE
fix(secu): remove unused http parameters in hostXML.php file for PHP5

### PR DESCRIPTION
**Pull request by sc979**
# Pull Request Template

## Description

remove unused http parameters in hostXML.php file.
cleaning file

**Fixes** # (none)

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [x] 2.8.x
- [ ] 18.10.x
- [ ] 19.04.x
- [ ] 19.10.x (master)

<h2> How this pull request can be tested ? </h2>

please contact me

#### Community contributors & Centreon team

- [x] I followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

#### Centreon team only

- [ ] I have made sure that the **unit tests** related to the story are successful.
- [ ] I have made sure that **unit tests cover 80%** of the code written for the story.
- [x] I have made sure that **acceptance tests** related to the story are successful (**local and CI**)

### CODE REVIEW

File: www/include/monitoring/status/Hosts/xml/hostXML.php [hunk 1]
```diff
@@ -190,60 +191,63 @@
 }
 
 if ($hostgroups) {
-    $rq1 .= " AND h.host_id = hhg.host_id AND hg.hostgroup_id IN ($hostgroups) AND hhg.hostgroup_id = hg.hostgroup_id";
+    $rq1 .= " AND h.host_id = hhg.host_id AND hg.hostgroup_id IN (" . $hostgroups . ") AND hhg.hostgroup_id = hg
+        .hostgroup_id";
```
Comment (callapa) [inline www/include/monitoring/status/Hosts/xml/hostXML.php]: Why did you put that line underneath it?
Split the line from the $hostgroups variable.

Comment (sc979) [reply www/include/monitoring/status/Hosts/xml/hostXML.php]: thanks, my IDE did it for me ;)

File: www/include/monitoring/status/Hosts/xml/hostXML.php [hunk 2]
```diff
@@ -71,40 +69,42 @@
...
+$num = $obj->checkArgument("num", $_GET, 0);
+$limit = $obj->checkArgument("limit", $_GET, 20);
+$search = $obj->checkArgument("search", $_GET, "");
+$order = isset($_GET['order']) && $_GET['order'] === "ASC"
+    ? "ASC"
+    : "DESC";
 
 $statusHost = $obj->checkArgument("statusHost", $_GET, "");
 $statusFilter = $obj->checkArgument("statusFilter", $_GET, "");
+$criticalityValue = $obj->checkArgument('criticality', $_GET, $obj->defaultCriticality);
```
Comment (sc979) [inline www/include/monitoring/status/Hosts/xml/hostXML.php]: criticality_id is renamed as in fact it's used in the query (line 163) to get a value (varchar) and not an id.

**Comment by sc979** [comment]
I'm sorry @callapa , but as the branch name is wrongly named I close this PR, to avoid odd behavior using Jira's hook :'(
Your comment has been taken in account in the "new" PR here : `https://github.com/centreon/centreon/pull/8013`

**Comment by sc979** [comment]
I'm sorry @callapa , but as the branch name is wrongly named I close this PR, to avoid odd behavior using Jira's hook :'(
Your comment has been taken in account in the "new" PR here : https://github.com/centreon/centreon/pull/8013

**Pull request closed without merging**